### ISSUE TITLE
NSF Biosketch

### ISSUE THREAD

**Issue by datalorax** [TysonStanley/biosketchr#3]
Next on the to-do list should probably be to add an NSF biosketch. Looks like there's a good latex template [here](https://www.overleaf.com/latex/templates/latex-template-for-nsf-style-biographical-sketch/mqrkmftcmwrq).

**Comment by TysonStanley** [comment]
Definitely! I don't think this won't be too difficult to get up and running. I can have a go at it later today (unless you wanted to get to it first).

**Comment by TysonStanley** [comment]
A quick glance at the template: would professional preparation and appointments be good to put in an Excel file like we did with NIH? It would be straightforward to get a function written up to format that.

We could also do that with the Collaborators and Other Affiliations section.

**Comment by datalorax** [comment]
Yeah I think that would be best. I like the idea of having all the "data" in spreadsheets that then gets pulled in and formatted.

**Comment by TysonStanley** [comment]
Great. I'll get started on that today.

**Comment by datalorax** [comment]
Perfect!

**Comment by TysonStanley** [comment]
Should be implemented at least to some degree. I wasn't running into any errors or warnings. As part of adding this, I adjusted some of the functions (to better differentiate between NIH and NSF). It may not be the best way of doing it so if you have better ideas on that, feel free to adjust!

**Comment by datalorax** [comment]
Great! I'll try to look at this later today.

**Comment by datalorax** [comment]
I took a look at this yesterday and overall I think it looks great! I have a few ideas I'd like to try to implement but this week is pretty tight. Will try to work on it as soon as possible though.

**Comment by TysonStanley** [comment]
No problem! Get to it when you can :)
